include/actor_types.h

```diff
@@ -5,6 +5,120 @@
 // Made this because there will likely be a lot of structs in this file that will be compiled into one eventually.
 // This keeps the other files cleaner.
 
+#define ACTOR_FALLING_ROCK        0x05
+#define ACTOR_BANANA              0x06
+#define ACTOR_GREEN_SHELL         0x07
+#define ACTOR_RED_SHELL           0x08
+#define ACTOR_YV_EGG              0x09 // YV -> Yoshi Valley
+#define ACTOR_PIRNHA_PLANT        0x0A
+#define ACTOR_ITEM_BOX            0x0C
+#define ACTOR_FAKE_ITEM_BOX       0x0D
+#define ACTOR_BANANA_BUNCH        0x0E
+#define ACTOR_TRAIN_ENGINE        0x0F
+#define ACTOR_TRAIN_COAL_CAR      0x10
+#define ACTOR_TRAIN_PASSENGER_CAR 0x11
+#define ACTOR_TRIPLE_GREEN_SHELL  0x15
+#define ACTOR_TRIPLE_RED_SHELL    0x16
+#define ACTOR_MRW_SIGN            0x17 // MRW -> Mario Raceway
+#define ACTOR_WS_SIGN             0x23 // WS -> Wario Stadium
+#define ACTOR_PADDLE_WHEEL        0x26 // Just the paddle wheel, or does it mean the entire boat?
+#define ACTOR_RR_CROSSING         0x27 // RR -> Railroad
+#define ACTOR_BLUE_SHELL          0x2A
+#define ACTOR_HAB_ITEM_BOX        0x2B // HAB -> Hot Air Baloon
+#define ACTOR_KIWANO_FRUIT        0x2D
+
+/*
+Used by the engine, coal car, and passenger car
+*/
+struct train_car {
```

Comment:
Do we really need a new struct for each object if they appear to use the generic actor struct?

Perhaps for modding purposes this makes sense?

Reply:
Maybe not? Most of the objects I've added structs for appear to have enough differences that a new struct type is useful for human readability (in my opinion at leat). Except maybe the FallingRock struct since that is a pretty "vanilla" object type compared to say, piranha plants.

Reply:
The piranha plants literally didn't match without its own struct.

Sounds reasonable to me, we'll go with your method.